aten/src/ATen/cpu/vec256/vsx/vsx_helpers.h

```diff
@@ -0,0 +1,332 @@
+#pragma once
+#include <cstdint>
+#include <c10/macros/Macros.h>
+#include <ATen/cpu/vec256/intrinsics.h>
+
+using vbool8   =  __attribute__((altivec(vector__))) __attribute__((altivec(bool__))) char;
```

Comment:
How about replacing `__attribute__((altivec(vector__)))` by `__vector` for readability?

Reply:
I had to do so because of GCC and default compilation. I opened the issue [advance-toolchain/issues/1861](https://github.com/advancetoolchain/advance-toolchain/issues/1861).

Reply:
Ouch, I see. Shame